src/services/utilServices/__tests__/SitesService.spec.js

```diff
@@ -24,18 +28,140 @@ describe("Resource Page Service", () => {
+      // Store the API key and set it later so that other tests are not affected
+      const currRepoCount = process.env.ISOMERPAGES_REPO_PAGE_COUNT
+      process.env.ISOMERPAGES_REPO_PAGE_COUNT = 3
+
+      const expectedResp = [
+        {
+          lastUpdated: repoInfo.pushed_at,
+          permissions: repoInfo.permissions,
+          repoName: repoInfo.name,
+          isPrivate: repoInfo.private,
+        },
+        {
+          lastUpdated: repoInfo2.pushed_at,
+          permissions: repoInfo2.permissions,
+          repoName: repoInfo2.name,
+          isPrivate: repoInfo2.private,
+        },
+      ]
+      mockIsomerAdminsService.getByUserId.mockImplementationOnce(() => null)
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [repoInfo, repoInfo2, adminRepo, noAccessRepo],
+      }))
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [],
+      }))
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [],
+      }))
+
+      await expect(service.getSites(mockSessionData)).resolves.toMatchObject(
+        expectedResp
+      )
+
+      expect(genericGitHubAxiosInstance.get).toHaveBeenCalledTimes(3)
+      process.env.ISOMERPAGES_REPO_PAGE_COUNT = currRepoCount
+      expect(process.env.ISOMERPAGES_REPO_PAGE_COUNT).toBe(currRepoCount)
+    })
+
+    it("Filters accessible sites for email user correctly", async () => {
+      // Store the API key and set it later so that other tests are not affected
+      const currRepoCount = process.env.ISOMERPAGES_REPO_PAGE_COUNT
+      process.env.ISOMERPAGES_REPO_PAGE_COUNT = 3
+
+      const expectedResp = [
+        {
+          lastUpdated: repoInfo.pushed_at,
+          permissions: repoInfo.permissions,
+          repoName: repoInfo.name,
+          isPrivate: repoInfo.private,
+        },
+      ]
+      mockIsomerAdminsService.getByUserId.mockImplementationOnce(() => null)
+      mockUsersService.findSitesByUserId.mockImplementationOnce(() => ({
+        site_members: [{ repo: { name: repoInfo.name } }],
+      }))
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [repoInfo, repoInfo2, adminRepo, noAccessRepo],
+      }))
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [],
+      }))
+      genericGitHubAxiosInstance.get.mockImplementationOnce(() => ({
+        data: [],
+      }))
+
+      await expect(
+        service.getSites(mockSessionDataEmailUser)
+      ).resolves.toMatchObject(expectedResp)
+
+      expect(mockIsomerAdminsService.getByUserId).toHaveBeenCalledWith(
+        mockIsomerUserId
+      )
+      expect(genericGitHubAxiosInstance.get).toHaveBeenCalledTimes(3)
+      process.env.ISOMERPAGES_REPO_PAGE_COUNT = currRepoCount
+      expect(process.env.ISOMERPAGES_REPO_PAGE_COUNT).toBe(currRepoCount)
+    })
+
+    it("Filters accessible sites for email user with no sitescorrectly", async () => {
```

Comment:
nit: add spacing between sites and correctly

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/480/commits/686087e5ee9a7cb687931ed23f0774783f633f9a